zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -499,6 +517,17 @@ fn snapshot_rpc_validateaddress(
     });
 }
 
+/// Snapshot `z_validateaddress` response, using `cargo insta` and JSON serialization.
+fn snapshot_rpc_z_validateaddress(
+    variant: &'static str,
+    z_validate_address: z_validate_address::Response,
+    settings: &insta::Settings,
+) {
+    settings.bind(|| {
+        insta::assert_json_snapshot!(format!("z_validate_address_{variant}"), z_validate_address)
+    });
+}
```

Comment:
To pass CI, this snapshot test will need new snapshot files:
https://github.com/ZcashFoundation/zebra/blob/0ee67d5d7cbcf1107a0ae94e212ccfac8f27b50d/zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs#L3-L6

Reply:
Done in [1c14599](https://github.com/ZcashFoundation/zebra/pull/6185/commits/1c145991e7354f25129fa32617c691dba118ddcd).